blocks/block_processor.go

```diff
@@ -5,27 +5,20 @@ import (
 	"crypto"
 	"fmt"
 
-	"github.com/alphabill-org/alphabill-explorer-backend/types"
 	abtypes "github.com/alphabill-org/alphabill/types"
+	exTypes "github.com/alphabill-org/alphabill-explorer-backend/types"
 )
 
 const (
 	DustBillDeletionTimeout    = 65536
 	ExpiredBillDeletionTimeout = 65536
 )
 
-type Store interface {
-	GetBlockNumber() (uint64, error)
-	SetBlockNumber(blockNumber uint64) error
-	AddTxInfo(txExplorer *types.TxInfo) error
-	SetBlockInfo(b *abtypes.Block) error
-}
-
 type BlockProcessor struct {
-	store Store
+	store exTypes.BillStore
```

Comment:
don't move interfaces to different packages, consumer interfaces in Go must be in the same namespace